drivers/serial/Kconfig.ns16550

```diff
@@ -53,6 +53,24 @@ config UART_NS16550_VARIANT_NS16950
 	  This enables support for 128-bytes FIFO and automatic hardware flow control.
 endchoice
 
+choice
+	prompt "UART Boot priority"
+	default UART_NS16550_PRE_KERNEL
+	help
+	  Select boot priority for UART
+
+config UART_NS16550_PRE_KERNEL
+	bool "ns16550 UART is defined PRE-KERNEL"
+	help
+	  This will set uart driver as pre-kernel
+
+config UART_NS16550_POST_KERNEL
+	bool "ns16550 UART is defined POST-KERNEL"
+	help
+	  This will set uart driver as post-kernel
```

Comment:
UART. Also, I'd either use "post-kernel" (human language), or "POST_KERNEL" (the preprocessor macro), but not "POST-KERNEL" (I don't think that exists?)